benchmark/single-source/DictionaryGroup.swift

```diff
@@ -39,9 +41,11 @@ class Box<T : Hashable> : Hashable {
 
 @inline(never)
 public func run_DictionaryGroupOfObjects(_ N: Int) {
-    let count = 20 * N
+  for _ in 1...N {
+    let count = 5_000
     let result = count / 10
     let dict = Dictionary(grouping: (0..<count).map { Box($0) }, by: { Box($0.value % 10) })
```

Comment:
Also, as the N get's balanced by the benchmark driver, why not make the count same between `DictionaryGroup` and `DictionaryGroupOfObjects`, so that their respective ratio directly reflects the change in work going from structs to classes?